_layouts/default.html

```diff
@@ -117,25 +132,33 @@ <h2 id="title">
         </ul>
       </nav>
       {%- endif -%}
-      <p>&copy;
+      <p class="about-footer {{condensed_class}}">&copy;
         {{ "now" | date: "%Y" }}</p>
       {%- if site.plainwhite.dark_mode -%}
-      <div>
+      <div class="about-footer {{condensed_class}}">
         <p>Dark Mode
           <i class="icon-moon"></i>
           <label class="switch">
-            <input type="checkbox" id="dark-mode-toggle">
+            <input type="checkbox" class="dark-mode-toggle">
             <span class="slider round" onclick="toggleDarkMode()"></span>
           </label>
         </p>
       </div>
-      <script type="text/javascript" src="{{ "/assets/js/darkmode.js" | relative_url }}"></script>
       {%- endif -%}
+      {% endcapture %}
+      {{ footer }}
     </section>
     <section class="content">
       {{ content }}
     </section>
+    <footer class="{{condensed_class}}">
+      {{ footer }}
+    </footer>
   </main>
+  {% if site.plainwhite.dark_mode %}
+  <script type="text/javascript" src="{{ "/assets/js/darkmode.js" | relative_url }}"></script>
```

Comment:
I suggest placing this higher in the DOM, specifically aboce `<section class="content">` which is generally the largest part of the HTML download.  That way reduces chances of the page flickering/flashing due to enablement of darkmode by making sure this remains executed early on, before any content is rendered.

Reply:
It has to be this far down to affect the toggle button in the footer.

Anyway, I think I found a way to get rid of the flicker completely: #83